vtex-apps/curbside-pickup
# [CURBPICKUP-4] Fixed setup in admin panel

What problem is this solving?

Bugfix for setup in the admin panel, the bug was showing the setup button even the workspace is already set up. 

How to test it?

Link the update and go to the curbside pickup admin page, see if the setup button shows. If the setup button shows for the first time, click it and then refresh. Now you shouldn't see the button that asks you to set up anymore.
